templates/exports/partials/run_history_table.html

```diff
@@ -44,6 +47,17 @@
         </a>
       </td>
       {% endif %}
+
+      {% if not is_partial_list %}
```

Comment:
this is intentional to not show it on this page because right now it looks all of the project runs uses the same config file.